slack/other channels

```diff
@@ -0,0 +1,13 @@
+Here are some of our other channels and why you might want to join them.
+
+#implementers - Those implementing the specification and questions about implementation details
+#specification - Potential changes to the JSON Schema specification
+#tests - Development of the official test suite
+#community - Development of the JSON Schema community and organisation
```

Comment:
Should we standardize on American spelling or British spelling?

```suggestion
#community - Development of the JSON Schema community and organization
```

Reply:
Reluctantly, I agree.
I think for all formal content, yes.
I'm not as strict with the blog articles, but maybe we should be?

Reply:
> Should we standardize on American spelling or British spelling?

I feel very left out by this comment :D